lib/internal/process/next_tick.js

```diff
@@ -5,6 +5,7 @@ exports.setup = setupNextTick;
 function setupNextTick() {
   const promises = require('internal/process/promises');
   const emitPendingUnhandledRejections = promises.setup(scheduleMicrotasks);
+  const MAX_TICKS = 1e4;
```

Comment:
A name like `kMaxCallbacksPerTick` is arguably more self-documenting - even if 'tick' is a misnomer by now, because...

> if breaking out in this way runs more nextTicks until the next turn of the event loop or if it runs another loop if more nextTicks remain/are scheduled

...the remaining ones run when the next call into the VM is made.  Anywhere you see a call to `MakeCallback()` in the C++ code, that's where nexttick callbacks run.